photutils/aperture/regions2aperture.py

```diff
@@ -0,0 +1,111 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+This module defines tools to convert regions.Region to Aperture.
+"""
+
+import astropy.units as u
+
+from photutils.aperture import (CircularAnnulus, CircularAperture,
+                                EllipticalAnnulus, EllipticalAperture,
+                                RectangularAnnulus, RectangularAperture,
+                                SkyCircularAnnulus, SkyCircularAperture,
+                                SkyEllipticalAnnulus, SkyEllipticalAperture,
+                                SkyRectangularAnnulus, SkyRectangularAperture)
+
+
+def regions2aperture(region_shape):
+    """
+    Convert a given `regions` shape to ``photutils`` aperture.
+
+    Parameters
+    ----------
+    region_shape : `regions.Region`
```

Comment:
```suggestion
    region : `regions.Region`
```

The term "shape" is a bit overloaded.  I try to reserve "shape" to mean numpy array shape in the photutils API.